src/tabs/__integ__/tabs.test.ts

```diff
@@ -223,13 +223,25 @@ test(
     await page.setWindowSize({ width: 500, height: 1000 });
     await page.click('#add-tab');
     await page.click(page.paginationButton('right', true));
+    await page.click(page.paginationButton('right', true));
```

Comment:
Why is it necessary to click once more in the right arrow now? Actually, it looks like the component scrolls a bit _further_ now than before, making the seventh tab visible with one single click whereas before it wasn't.

Reply:
The component actually scrolls slightly less than before, which is why this condition is necessary